src/internal/operations/backup.go

```diff
@@ -170,6 +170,22 @@ func (op *BackupOperation) Run(ctx context.Context) (err error) {
 			events.BackupID:  op.Results.BackupID,
 		})
 
+	defer func() {
+		op.bus.Event(
+			ctx,
+			events.BackupEnd,
+			map[string]any{
+				events.BackupID:   op.Results.BackupID,
```

Comment:
@ryanfkeepers - I took a look and all the fields referenced here seem same to use even if the backup fails - can you confirm that?